crates/primitives/src/kzg/generated.rs

```diff
@@ -0,0 +1,24 @@
+// @generated by build.rs from "src/kzg/trusted_setup.txt", do not modify manually.
+
+pub use c_kzg::{BYTES_PER_G1_POINT, BYTES_PER_G2_POINT};
+
+// Ensure that the build script constants are synced with the C bindings ones.
+const _: [(); BYTES_PER_G1_POINT] = [(); 48];
+const _: [(); BYTES_PER_G2_POINT] = [(); 96];
+
+pub const NUM_G1_POINTS: usize = 4096;
```

Comment:
can use `FIELD_ELEMENTS_PER_BLOB` const

Reply:
Can probably move this later in this task: https://github.com/bluealloy/revm/issues/686
But those fields are reintroduced to have preserve strict size at the time when points are moved to binary format (in build script).